### ISSUE TITLE
[FAILED] Failed unmounting /cdrom

### ISSUE THREAD

**Issue by TheNotary** [alvistack/vagrant-ubuntu#4]
I'm getting this error when copying the user-data and boot_command from [22.04](https://github.com/alvistack/vagrant-ubuntu/blob/master/packer/ubuntu-22.04-virtualbox/packer.json).  Anyone know what might be causing this problem?  

<img width="414" alt="Screenshot 2023-11-24 at 7 44 37 AM" src="https://github.com/alvistack/vagrant-ubuntu/assets/799247/19042ab9-a39e-44af-a4c5-6c7bf2be1a05">

**Comment by hswong3i** [comment]
But this is currently passed with CI: https://gitlab.com/alvistack/vagrant-ubuntu/-/jobs/5594174680

**Comment by hswong3i** [comment]
Wait, are you building with packer on Mac, but not on Linux?

**Comment by TheNotary** [comment]
I was building with virtualbox 7.0 (yes on mac indeed) and when removing the `quiet` switch I was able to observe that the underlying issue was with the network.  I eventually remembered needing some kind of workaround and sure enough, this is what happens when the `--nat-localhostreachable1` isn't applied.  

```
[ "modifyvm", "{{.Name}}", "--nat-localhostreachable1", "on" ]
```

I'll close this since that workaround is well demonstrated in your code (and is passing CI of course).  

Thanks btw for sharing this repo, I appreciate it!
